### ISSUE TITLE
PM classes become AM classes

### ISSUE THREAD

**Issue by Yustynn** [randName/sutd-timetable#18]
![image](https://user-images.githubusercontent.com/6380769/51588032-c1f3be00-1f1d-11e9-8ad8-e1a02bb05213.png)

I wanna do the Decision Theory class, but I don't 3am wanna do the Decision Theory class

**Comment by Yustynn** [comment]
Btw @randName if you include how to run your stuff in the readme, idm making a PR to fix it myself

**Comment by randName** [comment]
Yeah @charleswongzx mentioned this earlier. Seems to be a parsing issue since the db reflects this time. Check [src/plugins/timetable.js](../blob/master/src/plugins/timetable.js)

**Comment by Yustynn** [comment]
Can't debug without all your secrets for firebase and whatever else. But yeah just looking at the code idk what's wrong with it either.

Maybe the `date-fns` library is messed up. It's still in alpha after all
